### ISSUE TITLE
Buttons not mapping correctly from VR controllers

### ISSUE THREAD

**Issue by dpraul** [MichaelJW/DorsalVR#4]
Hi there! Been trying this out and something isn't quite right - Dolphin starts and I get the floating window after I launch a game, but the buttons on my controllers aren't passing through. When I check the controller config in Dolphin, I see that the Wiimote is set to the emulated controller and seems to be mapped to DorsalVR, but when I try pressing any of the buttons none none light up (although sometimes the trigger does?). When I tried the GameCubeExample, I had to manually select the controller to be from DorsalVR, but still none of the buttons worked except occasionally the trigger.

DorsalVR version: 2.0.1
Headset: HTC Vive (original) through SteamVR
Controllers: Valve Index controllers and HTC Vive wands (tried both)
Dolphin version: 5.0-14382 and 5.0-14596 (tried both)

I used the following config for Metroid Prime Trilogy, and I tried the `GameCubeExample` import with Metroid Prime 1:
```yaml
%YAML 1.1
---
imports:
- examples/WiiExample.yaml
dolphin:
  exePath: C:\games\Dolphin-x64\Dolphin.exe
  configDir: C:\Users\Dylan\Documents\Dolphin Emulator\Config
...
```

**Comment by MichaelJW** [comment]
Hey @dpraul, thanks for letting me know.

That's really strange; I would expect it to just show nothing if it ran into a problem. Definitely odd for just one button to work, and then only occasionally.

Would you mind downloading and running the debug build from [the Release page](https://github.com/MichaelJW/DorsalVR/releases/tag/v2.0.1) please? If there are any errors it should pop up a message you can see on your desktop. 

It should also create a log file called `Player.log` in the `/AppData/LocalLow/MichaelJW/DorsalVR/ `folder (one directory above the `/Config/` folder where your YAML files go); if you could attach that here it would be really useful. Thanks!

**Comment by dpraul** [comment]
Well, I guess I spoke too soon - I tried the debug build today and it worked, and then I reinstalled the regular build and it also worked. Not amazingly - the pointer was drifting like crazy, and Metroid Prime Trilogy crashed shortly after starting the campaign, but all the buttons to get through the menus worked!

An update was installed for SteamVR today - perhaps that fixed the issue. Regardless, I've attached the player.log files for fun.

[Debug Build - Player.log](https://github.com/MichaelJW/DorsalVR/files/6835324/Debug.Build.-.Player.log)
[Player.log](https://github.com/MichaelJW/DorsalVR/files/6835325/Player.log)

I'll reopen if I ever encounter this again.
